week5/solutions/cipher/cipher_text_message.py

```diff
@@ -1,7 +1,7 @@
 import os
 import string
 
-from Message import Message
+from message import Message
```

Comment:
Did you `git mv Message.py message.py`?

Reply:
I thought I did. But it appears I did not commit the change.